syntax = "proto3";

option java_multiple_files = true;
option java_package = "com.alexeyre.grpc.rental";
option java_outer_classname = "RentalServiceImpl";
option objc_class_prefix = "HLW";

package RentalService;

service RentalService {

	rpc rentalList(RentalListRequest) returns (stream RentalListResponse) {}
	rpc rentalBooking(stream RentalBookingRequest) returns (RentalBookingResponse) {}

	
}

message RentalListRequest{
	string value = 1;
}

message RentalListResponse{
	string result = 1;
}

message RentalBookingRequest{
	string text = 1;
}

message RentalBookingResponse{
	string rental = 1;
	string car = 2;
	string rentalDate = 3;
	string returnDate = 4;
}





